### ISSUE TITLE
For security reasons DTD is prohibited in this XML document.

### ISSUE THREAD

**Issue by Bluscream** [dnGrep/dnGrep#861]
![](https://i.imgur.com/uqCEXpz.png)

```
For security reasons DTD is prohibited in this XML document. To enable DTD processing set the DtdProcessing property on XmlReaderSettings to Parse and pass the settings into XmlReader.Create method.

Error opening the file: Value cannot be null.
Parameter name: encoding
```

https://stackoverflow-com.translate.goog/a/28459398?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US

**Comment by doug24** [comment]
Can you zip and upload that file, or a sample file that reproduces the problem?

**Comment by Bluscream** [comment]
> Can you zip and upload that file, or a sample file that reproduces the problem?

https://github.com/xlabs-mirror/xlabs-mirror.github.io/blob/main/index.html

**Comment by doug24** [comment]
Thanks for the file, and the pointer to the fix for the issue, that makes it easy.  I added that setting and it works.  I'll put this in the next release. PR: #862

**Comment by doug24** [comment]
Fixed in release v3.2.330

**Comment by Bluscream** [comment]
Thanks :3
